app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,70 @@
+class TweetsController < ApplicationController
```

Comment:
Great job splitting your actions among multiple controllers!

Reply:
Thank you for the great review. Just made those changes. Good to learn about ActiveRecord Validations and shortcuts for creating objects.